<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <widget class="QPushButton" name="gooo">
     <property name="text">
      <string>Играть!</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="heavyShot_1">
         <property name="text">
          <string>Крупный снаряд</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_2</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="midShot_1">
         <property name="text">
          <string>Средний снаряд</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_2</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="smallShot_1">
         <property name="text">
          <string>Лёгкий снаряд</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_2</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>22</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Игрок 1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>22</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Игрок 2 </string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QRadioButton" name="heavyShot_2">
         <property name="text">
          <string>Крупный снаряд</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_4</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="midShot_2">
         <property name="text">
          <string>Средний снаряд</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_4</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="smallShot_2">
         <property name="text">
          <string>Лёгкий снаряд</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_4</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="isHeavy_2">
         <property name="text">
          <string>Тяжёлый танк</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_3</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="isMiddle_2">
         <property name="text">
          <string>Средний танк</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_3</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="isLight_2">
         <property name="text">
          <string>Лёгкий танк</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_3</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="isHeavy_1">
         <property name="text">
          <string>Тяжёлый танк</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="isMiddle_3">
         <property name="text">
          <string>Средний танк</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="isLight_1">
         <property name="text">
          <string>Лёгкий танк</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QRadioButton" name="map_1">
       <property name="text">
        <string>Карта 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="map_2">
       <property name="text">
        <string>Карта 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="map_3">
       <property name="text">
        <string>Карта 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="map_4">
       <property name="text">
        <string>Карта 4</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup_3"/>
  <buttongroup name="buttonGroup_4"/>
 </buttongroups>
</ui>
